tests/functional/test_config_groups.py

```diff
@@ -79,6 +79,22 @@ def create_two_hosts(provider) -> Tuple[Host, Host]:
     return test_host_1, test_host_2
 
 
+@pytest.fixture()
+def cluster_with_components_on_one_host(
```

Comment:
Suggestion: since you have `FIRST_HOST` constant and here you bind both components on "first" host, so name this function to `cluster_with_components_on_one_host`.
The main profit is that when one's reading code where this fixture is used, it's clear that both components are bind to the "first" host, not "one of the hosts".

Reply:
renamed to cluster_with_components_on_first_host